Add `KEL LAN Tournament 3` wiki page and Add `KEL LAN Tournament` index page (#13302)
ppy/osu-wiki: wiki/Tournaments/KEL/3/en.md

4. All maps will be played with No Fail enabled, in addition to the appropriate mods.
5. Staff are prohibited from playing in the tournament except streamers, commentators, and GFX artists.

### Qualifier procedure

1. Players choose their lobby from the spreadsheet. They can participate in a lobby that has at least one empty slot.
   - Players notify hosts on the discord channel about their chosen lobby using a pinned prompt.
2. Last lobby for Qualifiers round cannot start after December 18th.
3. If a player does not show up to their scheduled Qualifiers lobby, they may play in any other lobby.
4. Each map will be played in order as they are shown on the mappool sheet. There will be one play through of the pool.
5. 90 second timers are used for players to ready up.
6. Top 21 players using avg sum of placements, will proceed to Bracket Stage.

### Bracket stage procedure

1. There are no warmups.
2. There are no restrictions on picks or bans.
3. No Fail is enforced on all maps.
4. Players will `!roll` in matches. The winner may choose between pick or ban order, and the other captain may choose the remaining order.

> 5. Players have 1 ban in Round of 32 & Round of 16, Quarterfinals & Semifinals and 2 bans throughout the rest of bracket stage.

Reviewer: but that's how the tournament went so uhh I don't know how to phrase it differently

> 5. Players have 1 ban in Round of 32 & Round of 16, Quarterfinals & Semifinals and 2 bans throughout the rest of bracket stage.

Reviewer 2: grammar was just weird

```suggestion
5. Players have 1 ban from the Round of 32 to Semifinals and 2 bans throughout the rest of bracket stage.
```

---
tags:
  - KEL3
  - KEL LAN
  - Polish LAN
---

# KEL LAN Tournament 3

![KEL LAN Tournament 3](/wiki/shared/news/2024-11-18-kel-lan-tournament-3/banner.jpg)

**KEL LAN Tournament 3** was a 1v1 double-elimination LAN tournament located in Gdańsk, Poland hosted by ::{ flag=PL }:: [myksmyks](https://osu.ppy.sh/users/6840526) and ::{ flag=PL }:: [flapczek](https://osu.ppy.sh/users/8210988). It was the third instalment of the KEL LAN Tournament series.

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2024-10-08/2024-12-02 |
| Qualifiers | 2024-12-04/2024-12-18 |
| Screening phase | 2024-12-18/2025-01-04 |
| Round of 32, Round of 16, Quarterfinals | 2025-02-08 |
| Semifinals, Finals, Grand Finals | 2025-02-09 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 40% of the prize pool, unique profile badge, Wooting UwU RGB |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 25% of the prize pool, Wooting UwU |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 15% of the prize pool |

> | "4th Place" | 12% of the prize pool |
> | "5th-6th Place" | 8% of the prize pool |

Reviewer 3: ```suggestion
| 4th Place | 12% of the prize pool |
| 5th-6th Place | 8% of the prize pool |
```
not sure if quotation marks are needed

![](img/badge.png "KEL LAN Tournament 3 winner badge")

## Organisation

| Position | Member(s) |
| :-- | :-- |

> | Organizers | ::{ flag=PL }:: [myksmyks](https://osu.ppy.sh/users/7696512), ::{ flag=PL }:: [flapczek](https://osu.ppy.sh/users/12840110) |

Reviewer 3: ```suggestion
| Organizers | ::{ flag=PL }:: [myksmyks](https://osu.ppy.sh/users/6840526), ::{ flag=PL }:: [flapczek](https://osu.ppy.sh/users/8210988) |
```

> | Production Team | ::{ flag=UK }:: [ilw8](https://osu.ppy.sh/users/11483110), ::{ flag=PL }:: [Nerfox](https://osu.ppy.sh/users/7976724), ::{ flag=PL }:: [flapczek](https://osu.ppy.sh/users/8210988), ::{ flag=SE }:: [memli](https://osu.ppy.sh/users/4027614), ::{ flag=PL }:: [myksmyks](https://osu.ppy.sh/users/6840526), ::{ flag=AU }:: [Fairy Bread](https://osu.ppy.sh/users/8306102), ::{ flag=FR }:: [ThePooN](https://osu.ppy.sh/users/718454), ::{ flag=IT }:: [Spazza17](https://osu.ppy.sh/users/3516241), ::{ flag=PL }:: [ahamarcel](https://osu.ppy.sh/users/11483110) |

Reviewer 3: ```suggestion
| Production Team | ::{ flag=GB }:: [ilw8](https://osu.ppy.sh/users/14167692), ::{ flag=PL }:: [Nerfox](https://osu.ppy.sh/users/7976724), ::{ flag=PL }:: [flapczek](https://osu.ppy.sh/users/8210988), ::{ flag=SE }:: [memli](https://osu.ppy.sh/users/4027614), ::{ flag=PL }:: [myksmyks](https://osu.ppy.sh/users/6840526), ::{ flag=AU }:: [Fairy Bread](https://osu.ppy.sh/users/8306102), ::{ flag=FR }:: [ThePooN](https://osu.ppy.sh/users/718454), ::{ flag=IT }:: [Spazza17](https://osu.ppy.sh/users/3516241), ::{ flag=PL }:: [ahamarcel](https://osu.ppy.sh/users/11483110) |
```

| Referees | ::{ flag=CL }:: [Bastaku](https://osu.ppy.sh/users/14351782), ::{ flag=KR }:: [Discord](https://osu.ppy.sh/users/16194858), ::{ flag=CL }:: [Isita](https://osu.ppy.sh/users/13973026), ::{ flag=SK }:: [Mavosiik](https://osu.ppy.sh/users/18927594), ::{ flag=PL }:: [Ninja Mejster](https://osu.ppy.sh/users/15827776), ::{ flag=US }:: [radioactive329](https://osu.ppy.sh/users/9826994), ::{ flag=HU }:: [Szofiianne](https://osu.ppy.sh/users/30979400), ::{ flag=DE }:: [TheHunter1](https://osu.ppy.sh/users/6496016), ::{ flag=RU }:: [-Hindeko](https://osu.ppy.sh/users/14220803), ::{ flag=PL }:: [czeczotka](https://osu.ppy.sh/users/14068656), ::{ flag=FI }:: [kimi321](https://osu.ppy.sh/users/6962961), ::{ flag=US }:: [Neutromint](https://osu.ppy.sh/users/25602812), ::{ flag=PL }:: [Whiter](https://osu.ppy.sh/users/16460997), ::{ flag=PL }:: [Wojtulas](https://osu.ppy.sh/users/12975155) |

> | Commentators | ::{ flag=ID }:: [BlankTap](https://osu.ppy.sh/users/10137131), ::{ flag=UK }:: [Bubbleman](https://osu.ppy.sh/users/5182050), ::{ flag=UK }:: [Damarsh](https://osu.ppy.sh/users/7465147), ::{ flag=UK }:: [Doomsday](https://osu.ppy.sh/users/18983), ::{ flag=PL }:: [Graf](https://osu.ppy.sh/users/4448391), ::{ flag=US }:: [im a fancy lad](https://osu.ppy.sh/users/4908650), ::{ flag=SK }:: [Mavosiik](https://osu.ppy.sh/users/18927594), ::{ flag=UK }:: [SadShiba](https://osu.ppy.sh/users/10747626), ::{ flag=IT }:: [Spazza17](https://osu.ppy.sh/users/3516241) |

Reviewer 3: ```suggestion
| Commentators | ::{ flag=ID }:: [BlankTap](https://osu.ppy.sh/users/10137131), ::{ flag=UK }:: [Bubbleman](https://osu.ppy.sh/users/5182050), ::{ flag=GB }:: [Damarsh](https://osu.ppy.sh/users/7465147), ::{ flag=GB }:: [Doomsday](https://osu.ppy.sh/users/18983), ::{ flag=PL }:: [Graf](https://osu.ppy.sh/users/4448391), ::{ flag=US }:: [im a fancy lad](https://osu.ppy.sh/users/4908650), ::{ flag=SK }:: [Mavosiik](https://osu.ppy.sh/users/18927594), ::{ flag=GB }:: [SadShiba](https://osu.ppy.sh/users/10747626), ::{ flag=IT }:: [Spazza17](https://osu.ppy.sh/users/3516241) |
```

- [Announcement post](https://osu.ppy.sh/home/news/2024-11-18-kel-lan-tournament-3-signups-now-open)
- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1987674)
- [Discord server](https://discord.gg/TSgMXsaZRR)
- [Livestream](https://www.twitch.tv/keltournaments)
- [YouTube](https://www.youtube.com/@KELtournaments)
- [Tournament bracket](https://challonge.com/kelosu3)

## Participants

|  | Country | Username |
| :-: | :-: | :-- |
| ::{ flag=PL }:: | **Poland** | [MALISZEWSKI](https://osu.ppy.sh/users/12408961) |
| ::{ flag=PL }:: | **Poland** | [hedayo](https://osu.ppy.sh/users/9323821) |
| ::{ flag=PL }:: | **Poland** | [Rafis](https://osu.ppy.sh/users/2558286) |
| ::{ flag=PL }:: | **Poland** | [Michni](https://osu.ppy.sh/users/3543130) |
| ::{ flag=NO }:: | **Norway** | [NINERIK](https://osu.ppy.sh/users/10549880) |
| ::{ flag=UA }:: | **Ukraine** | [sharytory](https://osu.ppy.sh/users/16139008) |
| ::{ flag=US }:: | **United States** | [im a fancy lad](https://osu.ppy.sh/users/4908650) |
| ::{ flag=PL }:: | **Poland** | [eniu](https://osu.ppy.sh/users/5472693) |

> | ::{ flag=UK }:: | **United Kingdom** | [Bubbleman](https://osu.ppy.sh/users/5182050) |
> | ::{ flag=UK }:: | **United Kingdom** | [Mahmood](https://osu.ppy.sh/users/7627844) |

Reviewer 3: ```suggestion
| ::{ flag=GB }:: | **United Kingdom** | [Bubbleman](https://osu.ppy.sh/users/5182050) |
| ::{ flag=GB }:: | **United Kingdom** | [Mahmood](https://osu.ppy.sh/users/7627844) |
```

| ::{ flag=PL }:: | **Poland** | [milosz](https://osu.ppy.sh/users/13108233) |
| ::{ flag=DE }:: | **Germany** | [ZeitFrost](https://osu.ppy.sh/users/11234706) |
| ::{ flag=PL }:: | **Poland** | [gnahus](https://osu.ppy.sh/users/12779141) |
| ::{ flag=UA }:: | **Ukraine** | [SHADOW FREAK](https://osu.ppy.sh/users/7587763) |
| ::{ flag=PL }:: | **Poland** | [Bartek22830](https://osu.ppy.sh/users/6404027) |
| ::{ flag=NO }:: | **Norway** | [HUNDUR](https://osu.ppy.sh/users/3145033) |
| ::{ flag=PL }:: | **Poland** | [MouseR](https://osu.ppy.sh/users/9855488) |
| ::{ flag=UA }:: | **Ukraine** | [RafGPio](https://osu.ppy.sh/users/13705417) |
| ::{ flag=PL }:: | **Poland** | [Tutka](https://osu.ppy.sh/users/8170022) |
| ::{ flag=PL }:: | **Poland** | [-BocheN-](https://osu.ppy.sh/users/18567604) |

> | ::{ flag=UK }:: | **United Kingdom** | [LilRaiN00](https://osu.ppy.sh/users/10903895) |

Reviewer 3: ```suggestion
| ::{ flag=GB }:: | **United Kingdom** | [LilRaiN00](https://osu.ppy.sh/users/10903895) |
```

  2. [Rohi feat. Kagamine Rin - Kakuzetsu Thanatos (YaMaDarknesss) [Collaboration of The End]](https://osu.ppy.sh/beatmapsets/2205703#osu/4669813)
  3. [seatrus - Blank society (Acylica) [Grand Trials]](https://osu.ppy.sh/beatmapsets/2226718#osu/4723710)
- Hard Rock
  1. [hasu - Warcry (melon boy) [THOUSAND PUNCH (kills instantly)]](https://osu.ppy.sh/beatmapsets/2281532#osu/4864391)
  2. [ELEMENTAS (A-One) - Yoru ga Oritekuru ~ Evening Star (Shurelia) [Last Word]](https://osu.ppy.sh/beatmapsets/2027939#osu/4225846)
  3. [AAAA + Umeboshi Chazuke - Cait Sith (bob) [Cat-sith]](https://osu.ppy.sh/beatmapsets/1848926#osu/3798091)
- Double Time
  1. [Shibayan feat. yana - Kare to Kanojo to Watashi no Hanashi (Elcheer) [Love]](https://osu.ppy.sh/beatmapsets/985131#osu/2382647)
  2. [sabi - true DJ MAG top ranker's song Zenpen (katagiri Remix) (Nathan) [Scub x Nathan's Insane]](https://osu.ppy.sh/beatmapsets/1357624#osu/2916949)
  3. [Nekomata Master+ - squall (Reisen Udongein) [Muya's Another]](https://osu.ppy.sh/beatmapsets/127772#osu/335800)
  4. [KikuoHana - Nobore! Susume! Takai Tou (Karen) [Expert]](https://osu.ppy.sh/beatmapsets/528821#osu/1121504)
- Tiebreaker

>   1. [nm-y vs. Kry.exe - Kaiju (Aerousea) [Primordial Disintegration Point]](https://osu.ppy.sh/beatmapsets/2284698#osu/4872864)

Reviewer 2: all tiebreakers must be bolded

Reviewer: done

  2. [technoplanet - Intuition (wakaba) [Sententialism]](https://osu.ppy.sh/beatmapsets/1824775#osu/3744646)
  3. [ZUTOMAYO - TAIDADA (Petal) [RESONATION]](https://osu.ppy.sh/beatmapsets/2270896#osu/4836943)
  4. [Cansol - 1,2,3,4! (Acylica) [geunssaem]](https://osu.ppy.sh/beatmapsets/2021545#osu/4210079)
- Hidden
  1. [UKRampage - Yumenoshima feat. mami (Gillstar) [Gill & Sharu's Dreamscape]](https://osu.ppy.sh/beatmapsets/1644513#osu/3356838)
  2. [Yasui Yousuke - YO-KAI Disco [MEIKAI ARRANGE VERSION] ([Keqing]) [aww shidd]](https://osu.ppy.sh/beatmapsets/1485831#osu/3046747)
- Hard Rock
  1. [Akatsuki Records - Treasure Cirno (Villdjack) [Stacked]](https://osu.ppy.sh/beatmapsets/2373198#osu/5124323)
  2. [Hagall x 152Hz - In midst of time feat. towatari (Cut Ver.) (sorciere) [Exploring the Unknown]](https://osu.ppy.sh/beatmapsets/998415#osu/2088253)
- Double Time
  1. [Asakura Hinata - Yukitoki (Amateurre) [Gelidium]](https://osu.ppy.sh/beatmapsets/1034179#osu/2257469)
  2. [Pendulum - Hold Your Colour (GRR SNARL GROWL) [Hold Your Growlr]](https://osu.ppy.sh/beatmapsets/2056846#osu/4880511)

## Ruleset

### General information

> 1. All matches will be played with ScoreV2 in Head to Head mode.

Reviewer 3: ```suggestion
1. All matches will be played with ScoreV2 in Head-to-Head mode.
```

2. Only 21 players will qualify for the bracket stage, chosen through Qualifier Stage.
3. Special guests that are announced as a player will be seeded with their BWS rank, placing them above qualified players.
4. All maps will be played with No Fail enabled, in addition to the appropriate mods.

> 5. Staff are prohibited from playing in the tournament with the exception of streamers, commentators, and GFX artists.

Reviewer 3: ```suggestion
5. Staff are prohibited from playing in the tournament except streamers, commentators, and GFX artists.
```

### Qualifier procedure

1. Players choose their lobby from the spreadsheet. They can participate in a lobby that has at least one empty slot.
   - Players notify hosts on the discord channel about their chosen lobby using a pinned prompt.
2. Last lobby for Qualifiers round cannot start after December 18th.
3. If a team does not show up to their scheduled Qualifiers lobby, they may play in any other lobby.

> 4. Each map will be played in order as they are shown on the mappool sheet. There will be one playthrough of the pool.

Reviewer 3: ```suggestion
4. Each map will be played in order as they are shown on the mappool sheet. There will be one play through of the pool.
```
